pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -1266,20 +1266,25 @@ func isConfigFile(filePath string) (useConfigMap bool, readonly bool, skip bool)
 func checkIsEmptyDir(filePath string) (bool, error) {
 	files, err := os.ReadDir(filePath)
 	if err != nil {
+		log.Errorf("ReadDir of %v failed !", filePath)
 		return false, err
 	}
 	if len(files) == 0 {
+		// is empty
```

Comment:
I don't think that we need these comments as the code is self-explanatory

Reply:
i've added to better understand because is not self explanatory... to understand where the bug...
but no problem i can remove...

Reply:
done

Reply:
Do you have a better approach for this?

Reply:
i've no problem about removing if you prefer, but i think in general is good to comment what is complex to understand... 

and because of searching about the bug i've been oblige to comment to better understand where the bug (and add logging)... 

now the bug is corrected, is not necessary anymore to maintain the comment... but in my way of coding i prefer more comment than no comment...

Reply:
I totally see your point, I suggest 2 possible alternatives:
- Keep comments but improve them by giving more context to the comment to make the reader understand better the code
- Add debug-level logs
wdyt? 
cc @cdrage

Reply:
i've removed is not a problem, take like it is. is not a problem. the most important is the resolution of the bug.